### PR TITLE
UI dev

### PR DESCRIPTION
**Pull request by vijeyash1**

### CODE REVIEW

File: dashboard/src/app/app.component.ts [hunk 1]
```diff
@@ -27,72 +29,92 @@ export class AppComponent implements OnDestroy {
   public currentProvider: any;
   public availableProviders: Array<any> = [
     {
-      label: 'Amazon Web Services',
```
Comment (narasago) [inline dashboard/src/app/app.component.ts:30]: Why has these lines been changed?

File: dashboard/src/app/app.component.ts [hunk 2]
```diff
@@ -27,72 +29,92 @@ export class AppComponent implements OnDestroy {
...
+    private ovhService: OvhService,
+    private azureService: AzureService
+  ) {
+    this.providers["aws"] = {
+      label: "Amazon Web Services",
+      value: "aws",
+      logo: "https://cdn.komiser.io/images/aws.png",
     };
 
-    this.providers['gcp'] = {
```
Comment (narasago) [inline dashboard/src/app/app.component.ts:59]: Please revert the changes which doesn't involve azure

File: dashboard/src/app/compute/azure/azure.component.ts [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+import 'jquery-mapael/js/maps/world_countries.js';
+
+import * as Chartist from 'chartist';
+import * as $ from 'jquery';
+import { Subscription } from 'rxjs';
+
+import { AfterViewInit, Component, OnDestroy, OnInit } from '@angular/core';
+
+import { AzureService } from '../../azure.service';
+import { DigitaloceanService } from '../../digitalocean.service';
```
Comment (narasago) [inline dashboard/src/app/compute/azure/azure.component.ts:12]: Azure component should not have reference to other platforms

File: go.mod [hunk 4]
```diff
@@ -0,0 +1,40 @@
+module github.com/mlabouardy/komiser
```
Comment (narasago) [inline go.mod:1]: Please don't add go mod

**Review by narasago — CHANGES_REQUESTED** [review]
Please check the comments and do the changes accordingly.

**Comment by vijeyash1** [comment]
In dashboard/src/app/app.component.ts
<https://github.com/narasago/komiser/pull/4#discussion_r739967207>:

> @@ -27,72 +29,92 @@ export class AppComponent implements OnDestroy {
   public currentProvider: any;
   public availableProviders: Array<any> = [
     {
-      label: 'Amazon Web Services',

Why has these lines been changed?
may be while saving alignment changed there.


In dashboard/src/app/compute/azure/azure.component.ts
<https://github.com/narasago/komiser/pull/4#discussion_r739970041>:

> @@ -0,0 +1,280 @@
+import 'chartist-plugin-tooltips';
+import 'jquery-mapael';
+import 'jquery-mapael/js/maps/world_countries.js';
+
+import * as Chartist from 'chartist';
+import * as $ from 'jquery';
+import { Subscription } from 'rxjs';
+
+import { AfterViewInit, Component, OnDestroy, OnInit } from ***@***.***/core';
+
+import { AzureService } from '../../azure.service';
+import { DigitaloceanService } from '../../digitalocean.service';

Azure component should not have reference to other platforms



am currently using digitaloceans compute dashboard as reference. in the
html page apart from active vms all others have reference to digital ocean.
for testing purpose i left it for now.
